### PR TITLE
Created Table.Wrapper

### PR DESCRIPTION
**Pull request by joeynimu**
Created a TableWrapper to wrap a table allowing it to be responsive.

### CODE REVIEW

File: src/components/Table/TableWrapper.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import Base from "../Base";
+
+const TableWrapper = styled(Base)`
+	width: 100%;
+	overflow-x: auto;
+	-webkit-overflow-scrolling: touch;
+	${prop("theme.TableWrapper")};
+`;
+
+export default as("tableWrapper")(TableWrapper);
```
Comment (diegohaz) [inline src/components/Table/TableWrapper.js]: It should render a `div`.

File: src/components/Table/TableWrapper.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+`Table.Wrapper` is composed from [Base](/components/primitives/base) . Table.Wrapper  renders by default as a `<tableWrapper>`.
+
+- `Table.Wrapper`
+
+```jsx
+import { Block } from "reakit";
+
+<Block overflowX="auto">
```
Comment (diegohaz) [inline src/components/Table/TableWrapper.md]: You can remove this `Block`. `TableWrapper` does the job.

File: src/components/Table/TableWrapper.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+`Table.Wrapper` is composed from [Base](/components/primitives/base) . Table.Wrapper  renders by default as a `<tableWrapper>`.
```
Comment (diegohaz) [inline src/components/Table/TableWrapper.md]: > as a `<div>`.

File: src/components/Table/TableWrapper.js (hunk 1 above)
Comment (joeynimu) [reply src/components/Table/TableWrapper.js]: noted. Is there a way to use `Base` and `Component` together? Can I do something like;
```
const TableWrapper = styled(Base)(Component) `
      width: 100%;
	overflow-x: auto;
	-webkit-overflow-scrolling: touch;
	${prop("theme.TableWrapper")}`;

```

Comment (diegohaz) [reply src/components/Table/TableWrapper.js]: Hmm. It'll not work. You should do this:
```jsx
const TableWrapper = styled(Base)`
  ...
`;

export default as("div")(TableWrapper);
```

Comment (joeynimu) [reply src/components/Table/TableWrapper.js]: aah...sorry, I should have had a look at the code base to begin with. Let me know how it looks now.

**Comment by diegohaz** [comment]
@joeynimu I like it. Thank you very much. Could you please allow edit from maintainers so I can tweak some things on the PR?

image

**Comment by joeynimu** [comment]
@diegohaz I think it's on, no? Let me know if you are not able to make edits.
screen shot 2018-07-18 at 7 34 12 pm

**Pull request closed without merging**